components/pkg-export-kubernetes/src/cli.rs

```diff
@@ -27,12 +31,19 @@ where
 }
 
 impl<'a, 'b> Cli<'a, 'b> {
+    /// Create a `Cli`
+    ///
+    /// # Arguments
+    ///
+    /// * `name` - The name of the CLI application to show in `--help' output.
+    /// * `about` - The long description of the CLi to show in `--help' output.
     pub fn new(name: &str, about: &'a str) -> Self {
         let app = docker::Cli::new(name, about).app;
 
         Cli { app: app }
     }
 
+    /// Convenient method to add all known arguments to the CLI.
```

Comment:
Convenience method? Otherwise you're singing your own praises :)

Reply:
It means the same thing. I know "convenience" is used typically but an adjective sounds more correct to me.

Reply:
This is getting OT but how can convenient and convenience mean the same thing?

Reply:
One is derived from the other (don't know which way around), one is an adjective and the other is a noun. Which is why it sounds wrong to me to use the noun form as an adjective.